src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1515,6 +1529,28 @@ private void OnCompileDotnetScript(MessageId msg_id, CancellationToken token)
             SendResponse(msg_id, Result.OkFromObject(new { }), token);
         }
 
+        private async Task<bool> OnContinueToLocation(MessageId sessionId, string script_id, int lineNumber, CancellationToken token)
+        {
+            DebugStore store = await RuntimeReady(sessionId, token);
+            ExecutionContext context = GetContext(sessionId);
+            Frame scope = context.CallStack.First<Frame>();
+            if (!SourceId.TryParse(script_id, out SourceId id))
+                return false;
+
+            var res = new List<SourceLocation>();
+
+            var sourceLocation = new SourceLocation(id, lineNumber, -1);
```

Comment:
Maybe get the column number from the location too, and if it's missing then default to -1?